src/guc.c

```diff
@@ -113,6 +113,15 @@ bool ts_shutdown_bgw = false;
 char *ts_current_timestamp_mock = NULL;
 #endif
 
+#ifdef TS_DEBUG
```

Comment:
I am wondering if the GUC / EVQ_Off setting should always be available to the user as a workaround when problems with the new implementation occur.

Reply:
It doesn't disable them but throws an error when they are used. I think it's a bad name, I will use 'require' instead of 'enable'. To disable vectorized quals, the user can disable the bulk decompression entirely, I think this is OK as a workaround.